syntax = "proto3";

service RiskAssessment {
    rpc AssessRisk(RiskRequest) returns (RiskResponse);
    rpc StreamRiskUpdates(Empty) returns (stream RiskResponse);
}

message RiskRequest {
    int64 customer_id = 1;
}

message RiskResponse {
    int64 customer_id = 1;
    string risk_level = 2;
}

message Empty {}

